Library/Homebrew/compilers.rb

```diff
@@ -6,6 +6,7 @@ module CompilerConstants
     "gcc-4.0"  => :gcc_4_0,
     "gcc-4.2"  => :gcc,
     "clang"    => :clang,
+    "brew_clang" => :brew_clang,
```

Comment:
I think it's fine; the version numbers only matter for `fails_with` invocations.